tests/pr_test.go

```diff
@@ -23,7 +23,12 @@ func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptio
 		Prefix:        prefix,
 		ResourceGroup: resourceGroup,
 		Region:        region,
+		TerraformVars: map[string]interface{}{
+			"profile": "cx2d-metal-96x192",
+			"zone":    "eu-gb-1",
```

Comment:
As suggested in the example, I would leave this as an integer only, that way its just added to the supplied region and this will make it easier to adjust in the future.